dev-support/hbase_nightly_yetus.sh

```diff
@@ -76,7 +76,11 @@ fi
 
 # For testing with specific hadoop version. Activates corresponding profile in maven runs.
 if [[ -n "${HADOOP_PROFILE}" ]]; then
-  YETUS_ARGS=("--hadoop-profile=${HADOOP_PROFILE}" "${YETUS_ARGS[@]}")
+  # Master has only Hadoop3 support. We don't need to activate any profile.
+  # The Jenkinsfile should not attempt to run any Hadoop2 tests.
+  if [[ "${BRANCH_NAME}" != "master" ]]; then
```

Comment:
I think here we should test for branch-2.x, not for !master, as feature branches should use the same solution with master.